cmd/containerd-shim-runhcs-v1/task_test.go

```diff
@@ -46,6 +46,14 @@ func (tst *testShimTask) GetExec(eid string) (shimExec, error) {
 	return nil, errdefs.ErrNotFound
 }
 
+func (tst *testShimTask) GetExecs() ([]shimExec, error) {
+	var execs []shimExec
+	for _, v := range tst.execs {
```

Comment:
doesnt `execs := append([]shimExec{}, tst.execs...)` work here?
or is it cause of typing?

Reply:
tst.execs is a map

Reply:
ohh, i missed that
thats embarrassing